content/Hardware Support/Nano Family/Nano-boards-that-can-be-powered-directly-with-3-3-V.md

```diff
@@ -3,59 +3,80 @@ title: "Nano boards that can be powered directly with 3.3 V"
+* The power you supply in this configuration must be pre-regulated to 3.3 V.
+* You can no longer use a USB connection to power the board.
+* You can no longer use a USB connection to upload sketches.
 
-Consequently, this configuration is mainly recommended for reducing power consumption for ready-to-deploy projects in which the board will be inactive for long periods of time.
+This configuration is mainly recommended ready-to-deploy projects where low power consumption is a priority.
 
 ---
 
-## Configure your board with 3.3 V power
+## Configure your board for 3.3 V power
 
-> **Note:** Sketches can not be uploaded via USB in this configuration!
+> **Warning:** Cutting the 3.3V pads will disable the USB connector! You won't be able upload sketches or power the board using USB until you restore the connection between the pads.
 
 1. Disconnect all power sources from your board.
 
-2. Identify the two 3.3V pads on the back of your board:
+2. Find the pads on the bottom of your board. They are outlined with a white, dotted line, and are labeled "3.3V".
+
+   ![The 3.3 V pads on different Arduino boards.](img/3.3V-pads-position.png)
 
-   ![The 3.3 V pads on different Arduino boards.](img/3.3V-pads.png)
+3. Use a sharp object (such as a hobby knife) to cut the trace (thin line of material) connecting the two pads:
 
-3. Cut the thin line of solder between the pads with a sharp object.
+   <div style="display: flex; flex-direction: row; flex-wrap: wrap;">
+     <figure style="text-align: center;">
+        <img style="height: 150px;" src="img/3.3V-pads-before.png" alt="">
+       <figcaption style="font-style: italic;">
+         Before cutting the trace.
+       </figcaption>
+     </figure>
+     <figure style="text-align: center;">
+        <img style="height: 150px;" src="img/3.3V-pads-after.png" alt="">
+        <figcaption style="font-style: italic;">
+          After cutting the trace.
+        </figcaption>
+     </figure>
+   </div>
 
-4. **Connect your regulated DC power source:**
+4. Connect your regulated DC power source:
 
-   * Connect your 3.3V+ power to the 3.3 V pin.
+   * Connect your 3.3V+ power to the 3V3/3.3V pin.
 
-   * Connect your power's ground a GND pin.
+   * Connect your power's ground to a GND pin.
 
-   ![Pin connections for direct 3.3 V power.](img/direct-power-connections.png)
+<figure style="width: 800px; margin: 0;">
+    <img src="img/direct-power-connections.png" alt="Circuit Diagram. UNO and Nano connected with digital and power pins.">
+    <figcaption style="text-align: center; font-style: italic;">Example: Battery connected to Nano RP2040 Connect.</figcaption>
+</figure>
 
 ---
 
 ## Restoring the default configuration
 
-To restore the solder jumper and return the board to its default configuration, apply a small amount of solder to short the two 3.3V pads.
+To restore the solder jumper and return the board to its default configuration, apply a small amount of solder to short the two 3.3V pads again.
```

Comment:
Changing on the line 6 the name for the pads might make sense to change it here aswell

Reply:
```suggestion
To return the board to its default configuration, apply a small amount of solder to short the two 3.3V pads again.
```